pages/create/index.js

```diff
@@ -0,0 +1,16 @@
+import SpotForm from "../../components/SpotForm.js";
+import Layout from "../../components/Layout.js";
+export default function CreateSpot({ addSpots }) {
+  function addSpotHandler(formData) {
+    const { id, Spotname, Address, Tasks, Equipment } = formData;
+    const newSpot = { id, Spotname, Address, Tasks, Equipment };
+
+    addSpots(newSpot);
+  }
```

Comment:
Not sure, if you then still need this function, as you could just pass `addSpotHandler` along to the `SpotForm` component.